crates/metadata/src/specs.rs

```diff
@@ -397,19 +428,19 @@ pub struct MessageSpec<F: Form = MetaForm> {
     ///
     /// In case of trait provided messages and constructors the prefix
     /// by convention in ink! is the label of the trait.
-    label: F::String,
+    pub label: F::String,
```

Comment:
Don't forget to remove all these `pub` modifiers to require use of builders

Reply:
almost forgot :monkey: thanks!

Reply:
`#[must_use]` is set for the `new<T>(ty: T)` function, however there is no other means to construct it anyways? There is no builder for `ReturnTypeSpec` (probably because there is only one fuild anyways). Should I implement one or can we rather just ditch the must_use?